src/pages/accessibility/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Document accessibility: getting from 5,000 to 0"
+authors:
+- Shannon Hahn
+- Shannon McHarg
+excerpt: "Our approach to making 5,000 documents on onrr.gov accessible."
+tags:
+- accessibility
+- section 508
+- inclusion
+- user experience
+- public service
+- government
+date: "2021-08-11"
+---
+
+When the Director of the Office of Natural Resources Revenue (ONRR) decided to ensure all documents on onrr.gov are Section 508 compliant, we thought it would be a reasonable project. We needed to come up with a plan to correct all the documents currently on the website and a plan for any new documents requested to be uploaded on the website. We started compiling [resources](https://github.com/ONRR/onrr.gov-site/wiki/Accessibility) to share with our colleagues to help them correct documents. With our Director’s support, we sent an agency-wide notification that we are no longer uploading any new documents on the website unless they pass the accessibility checks. Then we started going through the content inventory that [one of last year's interns](https://blog-nrrd.doi.gov/intern-experience/) had put together and found out there were over 3,000 documents currently on the site. That made the project seem daunting. We weren't sure we'd be able to get through all the documents in less than several years.
+
+## Prioritizing documents for compliance
+
+To start making sense of the pile of documents, we looked at our site's analytics to rank the documents by usage. We knew we should fix the most used documents first. We also found more documents that weren't in the content inventory. We put together a project plan that broke down the project into three phases and we emphasized that documents should be corrected by the owner since they are the subject matter experts of their own documents and accessibility is everyone’s responsibility. We were happy to learn that 70% of downloads over the course of a year were of the top 100 documents. We split those first 100 documents off as phase 1.
+
+At the beginning of phase 1, we ran accessibility reports on each document and assigned each document an owner. We sent each owner a list of the documents assigned to them for the phase along with the accessibility report. We also provided a link to our [wiki](https://github.com/ONRR/onrr.gov-site/wiki/Accessibility) with resources to make documents accessible. We also gave the owners a deadline about 8 weeks from the start date. One program area had the bulk of the documents, so we gave them a longer deadline of 20 weeks.
+
+We defined phase 2 as the next 100 most downloaded documents, which got us to 84% of downloaded documents. We sent each document owner their list of documents and accessibility reports. We gave document owners another 8 weeks to complete, phase 2, but it went faster than phase 1 because the document owners were now familiar with the making documents accessible.
```

Comment:
8 should be spelled out

```suggestion
We defined phase 2 as the next 100 most downloaded documents, which got us to 84% of downloaded documents. We sent each document owner their list of documents and accessibility reports. We gave document owners another eight weeks to complete, phase 2, but it went faster than phase 1 because the document owners were now familiar with the making documents accessible.
```